src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/ClassifiedCreateEventOperation.java

```diff
@@ -25,14 +23,13 @@ public ClassifiedCreateEventOperation(@Qualifier("default") CreateEventOperation
         this.classificationService = classificationService;
     }
 
-    @Transactional
     @Override
     public CaseDetails createCaseEvent(String caseReference,
                                        CaseDataContent content) {
         final CaseDetails caseDetails = createEventOperation.createCaseEvent(caseReference,
                                                                            content);
         return Optional.ofNullable(caseDetails)
                        .flatMap(classificationService::applyClassification)
-                        .orElseThrow(() -> new CaseNotFoundException(caseReference));
+                       .orElse(null);
     }
```

Comment:
Can we avoid returning null ?  It seems that a comuser of  createCaseEvent() would need to use the result to make a decision . would it be ok returning an  Optional .  Not mandatory .  Suggestion

Reply:
This was the existing logic. better to keep this logic for now